types/steam-user/index.d.ts

```diff
@@ -0,0 +1,5456 @@
+// Type definitions for steam-user 4.19
+// Project: https://github.com/DoctorMcKay/node-steam-user
+// Definitions by: vanitasboi <https://github.com/vanitasboi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.0
+// Enums generated from JS by: https://github.com/vanitasboi/JStoTSenum
+
+/// <reference types="node" />
+
+type SteamID = import('steamid');
```

Comment:
You can use this syntax IMO, which would make this more readable (about the intent):
```ts
import type SteamID = require('steamid');
import type ByteBuffer = require('bytebuffer');
```

Reply:
👍

Reply:
hat to bump the minimum typescript version to 4.2 for this to work ^^